renderer/components/dialog/icon.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default () => {
+  return (
+    <div>
+      <img src="https://getkap.co/static/favicon/kap.svg"/>
```

Comment:
This needs to change to the app icon. Wonder if there's a method to get the current app icon (stable/beta etc)

Reply:
Try the string `NSApplicationIcon` for https://www.electronjs.org/docs/api/native-image#nativeimagecreatefromnamedimageimagename-hslshift-macos

Reply:
<img width="76" alt="Screen Shot 2020-07-05 at 10 10 33 PM" src="https://user-images.githubusercontent.com/8822835/86549281-2eddf680-bf0d-11ea-992e-6869ce71418c.png">

This is what I get while in dev and built.

Also, this is the best quality I've been able to get using `scaleFactor`

Reply:
> Also, this is the best quality I've been able to get using scaleFactor

You should open an Electron issue about that. Most likely a bug.